security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -41,6 +41,15 @@ const (
 	// The Istio secret annotation type
 	IstioSecretType = "istio.io/key-and-cert"
 
+	// NamespaceManagedLabel (string, with namespace as value) and NamespaceOverrideLabel (boolean) contribute to determining
+	// whether or not a given Citadel instance should operate on a namespace. The behavior is as follows:
+	// 1) If NamespaceOverrideLabel exists and is valid, follow what this label tells us
+	// 2) If not, check NamespaceManagedLabel. If the value matches the Citadel instance's NS, then it should be active
+	// 3) If NamespaceManagedLabel nonexistent or invalid, follow enableNamespacesByDefault environment variable
```

Comment:
follow `enableNamespaceByDefault`, controlled by `CITALDE_XXX` environment variable.

the first is the field name, the second us user facing config, the actual env var name.